src/components/TeamNavbar/index.js

```diff
@@ -61,6 +61,17 @@ class TeamNavbar extends Component {
 
   handleDescriptionUpdate = (data: { description: string }) => this.props.onDescriptionUpdate(data);
 
+  descriptionText = () => {
+    if(this.teamIsLoaded()) {
+      const { description } = this.props.team;
+      return description ? description : 'Click here to edit description';
+    } else {
+      return '';
+    }
+  };
+
+  teamIsLoaded = () => this.props.team.id;
```

Comment:
Assuming every team has an ID, this checks that the team is loaded. Sorta janky, but works. Open to refactoring.